core/src/main/scala/quasar/repl/Main.scala

```
@@ -137,7 +137,7 @@ object Main {
 
       coreApi      <- CoreEff.interpreter.liftM[MainErrT]
       ephemeralApi =  CfgsErrsIO.toMainTask(MntCfgsIO.ephemeral) compose coreApi
-      _            <- mountAll[CoreEff](config.mountings) foldMap ephemeralApi
+      _            <- (mountAll[CoreEff](config.mountings) foldMap ephemeralApi).flatMapF(_.point[Task])
```

Yes. What was happening there is `mountAll foldMap ephemeralApi` was returning `String \/ Unit`, and then we were binding that to `_`, discarding the error in the process. Flatmap/pointing puts the error on the left in MainTask, which gets it logged and stops forward progress, which is what we want for now in the case that mounting fails.

The following commit applies the same change to the api server.